authmodule/src/main/java/com/accolite/team6/springsecurity/SpringSecurityConfig.java

```diff
@@ -0,0 +1,48 @@
+package com.accolite.team6.springsecurity;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.password.NoOpPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+@EnableWebSecurity
+public class SpringSecurityConfig extends WebSecurityConfigurerAdapter{
+
+	@Override
+	protected void configure(AuthenticationManagerBuilder auth) throws Exception{
+//		auth.inMemoryAuthentication()
+//		.withUser("Ayush")
+//		.password("password")
+//		.roles("USER")
+//		.and()
+//		.withUser("Malik")
+//		.password("password")
+//		.roles("ADMIN");
+	}
+	@Bean
+	public PasswordEncoder getPasswordEncoder() {
+		return NoOpPasswordEncoder.getInstance();
+	}
+	@Override
+	protected void configure(HttpSecurity http) throws Exception{
+//		http.authorizeRequests()
```

Comment:
Let's refrain ourself from adding commented out code unless it's really required and have proper commenting.

Reply:
Changed // with /**/ comments. Added details related to In-Memory Authentication